api/approval.go

```diff
@@ -37,11 +37,12 @@ type PRRApproval struct {
 	Number string `json:"kepNumber" yaml:"kep-number" validate:"required"`
 
 	// TODO: Need to validate these milestone pointers are not nil
-	Alpha      *PRRMilestone `json:"alpha" yaml:"alpha,omitempty"`
-	Beta       *PRRMilestone `json:"beta" yaml:"beta,omitempty"`
-	Stable     *PRRMilestone `json:"stable" yaml:"stable,omitempty"`
-	Deprecated *PRRMilestone `json:"deprecated" yaml:"deprecated,omitempty"`
-	Removed    *PRRMilestone `json:"removed", yaml:"removed,omitempty"`
+	Alpha              *PRRMilestone `json:"alpha" yaml:"alpha,omitempty"`
+	Beta               *PRRMilestone `json:"beta" yaml:"beta,omitempty"`
+	Stable             *PRRMilestone `json:"stable" yaml:"stable,omitempty"`
+	Deprecated         *PRRMilestone `json:"deprecated" yaml:"deprecated,omitempty"`
+	Removed            *PRRMilestone `json:"removed" yaml:"removed,omitempty"`
+	DeprecatedDisabled *PRRMilestone `json:"deprecatedDisabled" yaml:"deprecated-disabled,omitempty"`
```

Comment:
Super nit-pick, but maybe we want "deprecatedAnnounced" and "deprecatedDisabled" for congruence?

Reply:
Or deprecated->disabled->removed ?

Reply:
For compatibility reasons, I think deprecated->disabled->removed is better, because some KEPs already use `Deprecated` as milestone, and changing it to `deprecatedAnnounced` might confuse some users!